hazelcast-client/src/main/java/com/hazelcast/client/impl/statistics/Statistics.java

```diff
@@ -371,8 +371,19 @@ void fillMetrics(final StringBuilder stats, final ClientConnection ownerConnecti
             }
 
             for (Map.Entry<String, Gauge> entry : allMetrics.entrySet()) {
-                stats.append(STAT_SEPARATOR).append(entry.getKey()).append(KEY_VALUE_SEPARATOR);
-                entry.getValue().render(stats);
+                String key = entry.getKey();
+                stats.append(STAT_SEPARATOR).append(key).append(KEY_VALUE_SEPARATOR);
+
+                Gauge value = entry.getValue();
+                if (value == null) {
+                    value = metricsRegistry.newGauge(key);
```

Comment:
can a guage ever be null? I thought the MetricRegistry never return a null as a gauge?